charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -441,6 +445,11 @@ spec:
 {{- end }}
 {{- end }}{{/* end of authProxy.enabled */}}
       volumes:
+{{- if and (.Values.networking.useGlobalProxyNetworking) (not eq .Values.networking.proxy.config.trusted_ca_bundle "") }}
```

Comment:
I'm not sure this templating logic is necessary (or valid). Why can't we just do the original,
```yaml
{{- if and .Values.networking.useGlobalProxyNetworking .Values.networking.proxy.config.trusted_ca_bundle }}
```